folioreader/src/main/java/com/folioreader/ui/folio/activity/FolioActivity.java

```diff
@@ -207,6 +211,16 @@ public void onClick(View v) {
         }
     }
 
+    private void saveReceivedHighLights() {

```

Comment:
List<Highlight> need not to be passed in FolioActivity. It can be saved from FolioReader class..